src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -2816,7 +2818,8 @@ CreateBackgroundJob(const char *jobType, const char *description)
  */
 BackgroundTask *
 ScheduleBackgroundTask(int64 jobId, Oid owner, char *command, int dependingTaskCount,
-					   int64 dependingTaskIds[])
+					   int64 dependingTaskIds[], int nodesInvolvedCount, int32
```

Comment:
similarly, seems like missing a rebase?

Reply:
Let me fix the changed files. Not ready to be reviewed apparently. I will ping when ready.